docs/references/uvh5_memo.tex

```diff
@@ -0,0 +1,509 @@
+\item \textbf{Nblts}: \textit{int} The number of baseline-times (i.e., the
+  number of spectra) present in the data. Note that this value need not be equal
+  to Nbls $\times$ Ntimes. (\textit{Nblts})
+\item \textbf{Nfreqs}: \textit{int} The number of frequency channels in the
+  data. (\textit{Nfreqs})
+\item \textbf{Npols}: \textit{int} The number of polarization products in the
+  data. (\textit{Npols})
+\item \textbf{Ntimes}: \textit{int} The number of time samples present in the
+  data. (\textit{Ntimes})
+\item \textbf{Nspws}: \textit{int} The number of spectral windows present in the
+  data. (\textit{Nspws})
+\item \textbf{uvw\_array}: \textit{float} An array of the uvw-coordinates
+  corresponding to each observation in the data. This is a two-dimensional array of
+  size (Nblts, 3). (\textit{uvw\_array})
+\item \textbf{time\_array}: \textit{float} An array of the Julian Date
+  corresponding to the center of an integration. This is a one-dimensional array
+  of size Nblts. (\textit{time\_array})
+\item \textbf{integration\_time}: \textit{float} An array of the length of time
+  in seconds of an integration. This is a one-dimensional array of size
+  Nblts. (\textit{time\_array})
+\item \textbf{freq\_array}: \textit{float} An array of the frequencies stored in
+  the file in Hertz. This is a two-dimensional array of size (Nspws,
+  Nfreqs). (\textit{freq\_array})
+\item \textbf{channel\_width}: \textit{float} The width of frequency channels in
+  the file in Hertz. (\textit{channel\_width})
+\item \textbf{spw\_array}: \textit{int} An array of the spectral windows in the
+  file. This is a one-dimensional array of size Nspws. (\textit{spw\_array})
+\item \textbf{polarization\_array}: \textit{int} An array of the polarizations
+  contained in the file. This is a one-dimensional array of size Npols. Note
+  that the polarizations should be stored as an integer, and use the convention
+  defined in AIPS Memo 117. (\textit{polarization\_array})
+\end{itemize}
+
+\subsection{Optional Parameters}
+\label{sec:opt_params}
+\begin{itemize}
+\item \textbf{dut1}: \textit{float} DUT1 (google it). AIPS 117 calls it
+  ``UT1UTC''. (\textit{dut1})
+\item \textbf{earth\_omega}: \textit{float} Earth's rotation rate in degrees per
+  day. (\textit{earth\_omega})
+\item \textbf{gst0}: \textit{float} Greenwich sidereal time at midnight on
+  reference date. (\textit{gst0})
+\item \textbf{rdate}: \textit{string} Date for which GST0 (or whichever time
+  saved in that field) applies. (\textit{rdate})
+\item \textbf{timesys}: \textit{string} Time system. pyuvdata currently only
+  supports UTC. (\textit{timesys})
+\item \textbf{x\_orientation}: \textit{string} The orientation of the x-arm of a
+  dipole antenna. It is assumed to be the same for all antennas in the
+  dataset. For instance, ``E'' or ``East'' may be
+  used. (\textit{x\_orientation}).
+\item \textbf{telescope\_name}: \textit{string} The name of the telescope used
```

Comment:
Interesting that this is optional. I think that's because you've made "instrument" required and I think that makes sense for telescopes where those two field are the same (e.g. MWA, PAPER, HERA), but at least in uvfits and miriad, the telescope name is in the `telescop` field and then `instrume` describes the receiver or backend.

Reply:
ah, fair point. I think I'll make it a required parameter, especially because we have those self-consistency checks conditioned on the telescope name.